hotham/src/engine.rs

```diff
@@ -85,23 +86,44 @@ impl<'a> EngineBuilder<'a> {
             .expect("!!FATAL ERROR - Unable to initialize renderer!");
         let gui_context = GuiContext::new(&vulkan_context);
 
+        // Initialize the world with our "tracking" entities, the stage and the HMD.
+        let mut world = hecs::World::default();
+        let (stage_entity, hmd_entity) = create_tracking_entities(&mut world);
```

Comment:
Since we treat `Stage` as a first class entity (ie. our renderer knows about it!) it makes sense to have the engine create this on startup and keep a reference to the entity to avoid having to keep query for the stage all the time.